### PR TITLE
SG-36830 Upgrade Twisted -  automated

### PR DESCRIPTION
**Pull request by carlos-villavicencio-adsk**
Generated from #270

### CODE REVIEW

File: resources/python/src/3.7/pkgs.zip
Comment (julien-lang) [inline resources/python/src/3.7/pkgs.zip:1]: Why do we remove this file?

File: resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so
Comment (julien-lang) [inline resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: Why do we remove this file?

Also what about Windows and macOS?

Comment (carlos-villavicencio-adsk) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: I don't think this was removed. Just updated.

File: resources/python/src/3.7/pkgs.zip
Comment (carlos-villavicencio-adsk) [reply resources/python/src/3.7/pkgs.zip:1]: I don't think this was removed. Just updated.

Comment (julien-lang) [reply resources/python/src/3.7/pkgs.zip:1]: Yes, indeed. Sorry about that!

File: resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so
Comment (julien-lang) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: Ok. What about mac and Win?

Comment (julien-lang) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: The fact that the linux libs was updated but not the Windows or macOS one is suspicious. So I'm raising a question: is this expected or is it possible there is a glitch in the process?

Comment (carlos-villavicencio-adsk) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: It seems to work as expected on Linux Rocky 8. 
image

Comment (julien-lang) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: Sorry if I am not clear. Trying to 🤔!

What I mean is: why don't we have the Windows/macOS libs updated too? But only the Linux one? That does not make sense, does it?

Comment (carlos-villavicencio-adsk) [reply resources/python/bin/3.11/linux/zope/interface/_zope_interface_coptimizations.cpython-311-x86_64-linux-gnu.so:1]: I don't really know.

**Review by julien-lang — APPROVED** [review]
LGTM!

**Pull request merged**